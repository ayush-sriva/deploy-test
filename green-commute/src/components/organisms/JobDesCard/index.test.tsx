import ReactTestRenderer from 'react-test-renderer';
import JobDescCard from '.';
import HP from '../../../../public/assets/company-logos/hp.svg';

describe('Job Description Test', () => {
  it('renders the job description', () => {
    const tree = ReactTestRenderer.create(
      <JobDescCard
        companyLogo={HP}
        company={'HP'}
        title={'Developer'}
        location={'Hyderabad'}
        postedDate={'12min ago'}
        onClickHandler={() => {}}
      ></JobDescCard>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
