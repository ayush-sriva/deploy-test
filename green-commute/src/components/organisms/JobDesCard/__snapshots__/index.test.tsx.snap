// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Job Description Test renders the job description 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root css-phpnom-MuiPaper-root-MuiCard-root"
>
  <div
    className="MuiCardHeader-root css-so20wf-MuiCardHeader-root"
  >
    <div
      className="MuiCardHeader-avatar css-1ssile9-MuiCardHeader-avatar"
    >
      <img
        alt="company_logo"
        className="css-dejdd"
        src="hp.svg"
      />
    </div>
    <div
      className="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
    >
      <span
        className="MuiTypography-root MuiTypography-body2 MuiCardHeader-title css-et1ao3-MuiTypography-root"
      >
        <div
          className="MuiBox-root css-0"
        >
          <h6
            className="MuiTypography-root MuiTypography-subtitle1 makeStyles-title-1 css-10wpov9-MuiTypography-root"
          >
            Developer
          </h6>
          <span
            className="MuiTypography-root MuiTypography-caption makeStyles-location-3 css-1sn4lm3-MuiTypography-root"
          >
            HP
          </span>
        </div>
      </span>
      <span
        className="MuiTypography-root MuiTypography-body2 MuiCardHeader-subheader css-83ijpv-MuiTypography-root"
      >
        <div
          className="MuiBox-root css-0"
        >
          <span
            className="MuiTypography-root MuiTypography-caption makeStyles-location-3 css-1sn4lm3-MuiTypography-root"
          >
            Hyderabad
          </span>
          <br />
          <span
            className="MuiTypography-root MuiTypography-captionBold makeStyles-location-3 css-37jdci-MuiTypography-root"
          >
            12min ago
          </span>
        </div>
      </span>
    </div>
    <div
      className="MuiCardHeader-action css-sgoict-MuiCardHeader-action"
    >
      <div
        className="css-u1w75p"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-yoei7b-MuiSvgIcon-root-MuiSvgIcon-root"
          data-testid="SVGIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    className="MuiCardActions-root MuiCardActions-spacing css-vb1tq8-MuiCardActions-root"
  >
    <button
      className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeSmall MuiButton-outlinedSizeSmall MuiButtonBase-root  css-1k23hlb-MuiButtonBase-root-MuiButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiTypography-root MuiTypography-captionBold css-37jdci-MuiTypography-root"
      >
        Save
      </span>
    </button>
    <button
      className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButtonBase-root  css-11qr2p8-MuiButtonBase-root-MuiButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="submit"
    >
      <span
        className="MuiTypography-root MuiTypography-captionBold css-37jdci-MuiTypography-root"
      >
        Apply
      </span>
    </button>
  </div>
</div>
`;
